apps/console/src/app/components/prompts/PromptTester.tsx

```diff
@@ -141,7 +141,7 @@ export const PromptTester = () => {
                 <Col span={12}>
                   <Typography.Title level={3}>Request</Typography.Title>
                   <div style={{ background: "#000000", padding: 10 }}>
-                    <PromptEditor value={result.interpolatedContent} />
+                    {/* <PromptEditor value={result.interpolatedContent} /> */}
```

Comment:
I imagine we want to get rid of this

Reply:
If we don't use the Prompt Tester anymore, maybe it's best to remove it? We can always refer to it using Git history if we need to. Just to make it easier for contributors who see this component.